backend/tracim_backend/lib/mail_notifier/sender.py

```diff
@@ -63,12 +63,17 @@ def connect(self):
         if not self._smtp_connection:
             log = "Connecting to SMTP server {}"
             logger.info(self, log.format(self._smtp_config.server))
-            # TODO - G.M - 2019-01-29 - Support for SMTP SSL-only port connection
-            # using smtplib.SMTP_SSL
-            self._smtp_connection = smtplib.SMTP(self._smtp_config.server, self._smtp_config.port)
+            if self._smtp_config.implicit_ssl:
```

Comment:
why do you name it `IMPLICIT_SSL` while the smtplib name it `SMTP_SSL`

Reply:
Because there is explicit and implicit ssl, we can use ssl/tls in explicit mode (aka startssl) and it's the normalized method.

Reply:
i check flask plugin which does equivalent mecanism and the option name are worst : "use_ssl", "use_tls" (tls for startssl and ssl for encrypted smtp (in fact ssl is not used anymore and all tls is used anywhere for encryption)

Reply:
default tracim behaviour is trying explicit tls encryption using starttls if we do use a new option name "use_ssl" it will make thing that defaut tracim behavior is unencrypted which is not true.

By the way, we probably, should make defaut behavior configurable to avoid "uncorrect unencrypted connexion when we expect starttls but this is out of the scope of this fix and it's complicated to find a solution which either does work and do not break old behaviour.

Reply:
we need a new issue for refactor of these parameters in next major version. The point is ok for now

Reply:
Issue #2851 created about that